extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -64,15 +69,25 @@ class AmpStickyAd extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
-    this.isExperimentOn_ = isExperimentOn(this.getWin(), TAG);
+    // this.isExperimentOn_ = isExperimentOn(this.getWin(), TAG);
     if (!this.isExperimentOn_) {
       dev.warn(TAG, `TAG ${TAG} disabled`);
       return Promise.resolve();
     }
+    // Reschedule layout for ad if layout sticky-ad again.
+    if (this.visible_) {
+      this.updateInViewport(this.ad_, true);
+      this.scheduleLayout(this.ad_);
+    }
     return Promise.resolve();
   }
 
   /** @override */
+  unlayoutCallback() {
```

Comment:
It's for switching viewers without dismiss sticky-ad. I want it to layout again when switch back.